docs/.templates/cli.md

```diff
@@ -63,7 +65,147 @@ If you have entered a command to the `build.beforeBuildCommand` property, this o
 
 {icon}
 
-[Tauri Icon Guide](../guides/features/icons.md)
+For more information, check out the complete [Tauri Icon Guide](../guides/features/icons.md).
+
+## `completions`
+
+<Command name="completions" />
+
+{completions}
+
+The Tauri CLI can generate shell completions for Bash, Zsh, PowerShell and Fish. To write the completions script to a file, run tauri completions --shell [bash|zsh|powershell|fish] > outputFile.
```

Comment:
You mention Fish but I'm not seeing a section below for it

Reply:
probably because nobody knows how to do that with fish 😅 i think Lucas added that because clippy_complete says it supports it.

Gonna see if i find something

Reply:
I didn't even test completions on Fish tbh (but it looks alright)... I still think a Fish is that thing that lives in the ocean

Reply:
so far i only make fish show the completions once, but i think that was a bug somehow 😅 The issue is that the file name must be the same as the command to make fish pick it up so i'm a bit lost right now.

Looks like we'll have to name the file cargo.fish pnpm.fish etc which is super annoying. I guess i'll change the command to append to the file instead of moving/overwriting but that would break completion files from one of the other sources from the link above because of this [issue](https://togithub.com/fish-shell/fish-shell/issues/6814); what a mess.

Unless someone has an idea rn, I tend to just leave that unresolved for now and open a "help wanted" issue to see if someone with more fish experience can help out because i can't find anything helpful on the internet here.

Reply:
(~~Yeah, the one who requested fish support in clap_complete~~😂)

Reply:
https://github.com/tauri-apps/tauri-docs/issues/1288